libs/ml/include/ml/layers/PRelu.hpp

```diff
@@ -99,6 +99,15 @@ class PRelu : public SubGraph<T>
   }
 
   static constexpr char const *DESCRIPTOR = "ParametricRelu";
```

Comment:
presumably we can ditch this and just have the Descriptor function?

Reply:
I believe yes, but for now ::DESCRIPTOR is used in a template code 

template <typename OperationType>
bool Graph<TensorType>::UpdateVariableName(
...
  std::string op_descriptor = (OperationType::DESCRIPTOR);

so I guess we will ditch both DESCRIPTOR and OpType fields and replace them by virtual polymorphic functions after refactoring of GraphBuilder and that code above.